core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournalUnexpectedError.scala

```diff
@@ -1,3 +1,7 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
Not up to 2019?

Reply:
I won't change that in this PR but needs updating

Reply:
Ah, it's the plugin that did it?

Reply:
Yup, we're do out of date: https://github.com/akka/akka-persistence-cassandra/blob/master/build.sbt#L53

Will raise a separate PR to fix that